src/components/TransferFormComponents/TransferAmountTextField.jsx

```diff
@@ -65,7 +65,7 @@ export default function TransferAmountTextField (props: Props) {
     const decimals = getCryptoDecimals(cryptoType)
     if (
       !validator.isFloat(value, {
-        min: 0.001,
+        min: 0.0001,
```

Comment:
if you send 0.0001  BTC to an escrow address, and if tx fee is high it could cause insufficient fund sometime when transfer out of escrow.

Reply:
fixed